package.json

```diff
@@ -2,18 +2,19 @@
   "name": "spectacle",
   "version": "5.5.0",
   "description": "ReactJS Powered Presentation Framework",
+  "source": "src/components/index.js",
   "main": "lib/index.js",
   "module": "es/index.js",
   "types": "index.d.ts",
   "scripts": {
-    "build": "",
     "lint": "eslint --ignore-path .gitignore .",
     "lint-fix": "",
     "prettier": "prettier \"**/*.{js,json,ts,css,md}\"",
     "prettier-fix": "npm run prettier -- --write",
     "deploy": "",
     "start": "builder run --env \"{\\\"BABEL_ENV\\\":\\\"esm\\\"}\" build-wds-base -- --port=3000 --hot",
```

Comment:
If we're going full rollup and have buy-in from Spectacle team + @kale-stew let's go ahead and remove all `webpack` deps + build steps and replace `start` script here with a rollup version.

Reply:
Will do, I wasn't sure we wanted to remove + replace `webpack` in a future PR or within this piece of work. Happy to do it here.

Reply:
If it’s fast (like an hour or less), let’s do it here. If not separate pr. I’d like us to not have two builds for any longer than we need and I’d like the rewrite team to dogfood dev in rollup if there’s buy in immediately to flush out any dev issues early. Thanks!

Reply:
If it’s fast (like an hour or less), let’s do it here. If not separate pr. I’d like us to not have two builds for any longer than we need and I’d like the rewrite team to dogfood dev in rollup if there’s buy in immediately to flush out any dev issues early. Thanks!

Reply:
I've had a crack at replacing webpack dev server with rollup plugins `serve` and `livereload` - accessible via `yarn start`.
I still need to clean up & comment the config a bit but it's probably in a fairly usable state now. Would be good to get some of the team trying out & testing the workflow.